build-yahm-lxc.sh

```diff
@@ -0,0 +1,158 @@
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+
+echo -n "$(timestamp) [LXC] [openHABian] Setting default password for openhabian user..."
+echo openhabian:openhabian | lxc-attach -n openhabian -- chpasswd &>> /var/log/yahm/openhabian_install.log
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+
+# wait to get ethernet connection up 
+sleep 5
+
+echo -n "$(timestamp) [LXC] [openHABian] Cloning openhabian repository..."
+lxc-attach -n openhabian -- git clone -b master https://github.com/openhab/openhabian.git /opt/openhabian &>> /var/log/yahm/openhabian_install.log
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+
+echo -n "$(timestamp) [LXC] [openHABian] Linking openhabian configuration utility to /usr/bin..."
+lxc-attach -n openhabian -- ln -sfn /opt/openhabian/openhabian-setup.sh /usr/bin/openhabian-config &>> /var/log/yahm/openhabian_install.log
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+
+echo -n "$(timestamp) [LXC] [openHABian] Creating openhabian default configuration..."
+echo -e "hostname=openHABian\nusername=openhabian\nuserpw=openhabian\ntimeserver=0.pool.ntp.org\nlocales='en_US.UTF-8 de_DE.UTF-8'\nsystem_default_locale=en_US.UTF-8\ntimezone=Europe/Berlin" > /var/lib/lxc/openhabian/rootfs/etc/openhabian.conf &>> /var/log/yahm/openhabian_install.log
+if [ $? -eq 0 ]; then echo "OK"; else echo "FAILED"; fail_inprogress; fi
+	
+if [ $ARCH = "arm64" ]
+then
+	lxc-attach -n openhabian -- dpkg --add-architecture armhf
+fi	
+
+echo  "$(timestamp) [LXC] [openHABian] Starting openhabian installation, this can take a lot of time....."
+lxc-attach -n openhabian -- /opt/openhabian/openhabian-setup.sh unattended 
+
+if [ $? -eq 0 ]
+then 
+	echo  -e "\n$(timestamp) [GLOBAL] [openHABian] Installation Done.\n"
+else 
+	echo "FAILED"; fail_inprogress; 
+fi
+
+
+# Geting some IP informations
+YAHM_LXC_IP=$(lxc-info -i -n ${LXCNAME} | awk '{print $2}')
+OH_LXC_IP=$(lxc-info -i -n openhabian | awk '{print $2}')
+
+echo "$(timestamp) [INFO] Homematic  IP: ${YAHM_LXC_IP}"
+echo "$(timestamp) [INFO] OpenHABian IP: ${OH_LXC_IP}"
+echo -e "\n"
+
+echo  "$(timestamp) [LXC] [openHABian] Setup CCU2 Binding inside openHABian"
+OH_CONF_DIR=/var/lib/lxc/openhabian/rootfs/etc/openhab2
+sed -i $OH_CONF_DIR/services/addons.cfg -e 's/^#binding.*$/binding=homematic/'
+echo "Bridge homematic:bridge:yahm [ gatewayAddress='${YAHM_LXC_IP}' ]" > $OH_CONF_DIR/things/yahm-homematic.things
+lxc-attach -n openhabian  -- chown openhab:openhab /etc/openhab2/things/yahm-homematic.things
+
+echo  "$(timestamp) [LXC] [openHABian] Starting openHAB2 Service."
+lxc-attach -n openhabian -- systemctl start openhab2.service
+
+echo -e "\n"
+
+echo "$(timestamp) [INFO] OpenHABian Login URL: http://${OH_LXC_IP}:8080"
+echo "$(timestamp) [INFO] OpenHABian Console Login: lxc-attach -n openhabian"
```

Comment:
Please add a newline at the end of the file

Reply:
In whole script, please make sure that "openHAB" and "openHABian" are the used spellings